ivre/web/app.py

```diff
@@ -582,6 +583,17 @@ def get_flow():
     count = query.get("count", False)
     orderby = query.get("orderby", None)
     timeline = query.get("timeline", False)
+    try:
+        before = datetime.datetime.strptime(query.get("before", None),
```

Comment:
`, None` is not useful here. Also, maybe it could be good to have a debug / warning message when before is provided incorrectly.
```Python
    try:
        before = datetime.datetime.strptime(query["before"], "%Y-%m-%d %H:%M")
    except KeyError:
        before = None
    except ValueError:
        # XXX warning
        before = None
```